openreview/profile/management.py

```diff
@@ -219,4 +293,105 @@ def set_archive_invitations(self):
                     }
                 }
             }
-        ))                       
+        ))
+
+    def set_merge_profiles_invitations(self):
+
+        content = {
+            'email': {
+                'order': 1,
+                'description': 'email of the user making the request.',
+                'value-regex': '([a-z0-9_\-\.]{1,}@[a-z0-9_\-\.]{2,}\.[a-z]{2,},){0,}([a-z0-9_\-\.]{1,}@[a-z0-9_\-\.]{2,}\.[a-z]{2,})',
+                'required': False                
+            },
+            'left': {
+                'order': 2,
+                'description': 'Username or email that want to be merged.',
+                'value-regex': '.*',
+                'required': True
+            },
+            'right': {
+                'order': 3,
+                'description': 'Username or email that want to be merged.',
+                'value-regex': '.*',
+                'required': True
+            },
+            'comment': {
+                'order': 4,
+                'value-regex': '[\\S\\s]{1,5000}',
+                'description': 'Reason why you want to delete your name.',
```

Comment:
Should we change this to: `Reason why you want to merge your profiles.` (though I'm sure everyone wants to merge because they have multiple profiles). 
Maybe just `Any additional information about the merge.`

Reply:
good idea, @xkopenreview is the UI using this description?

Reply:
no currently the label of input is fixed.